fastlane/Deliverfile

```diff
@@ -7,18 +9,28 @@
 ########################################
 
 # This folder has to include one folder for each language
-screenshots_path "./screenshots/"
-app_identifier "com.automattic.SimplenoteMac"
+screenshots_path './screenshots/'
+app_identifier 'com.automattic.SimplenoteMac'
 
 # Make sure to update these keys for a new version
-app_version "2.12"
+app_version '2.12'
 
-privacy_url({
-  'default' => 'https://simplenote.com/privacy/',
-})
+privacy_url(
+  'default' => 'https://simplenote.com/privacy/'
+)
 
-copyright('2019 Automattic')
+copyright('2021 Automattic')
 
 skip_binary_upload true
 overwrite_screenshots true
 phased_release true
+
+# We are using the API key to upload the metadata to ASC via Deliver, but, as
+# the tool reports:
+#
+# > [!] Precheck cannot check In-app purchases with the App Store Connect API
+# > Key (yet). Exclude In-app purchases from precheck, disable the precheck step
+# > in your build step, or use Apple ID login
+#
+# Simplenote iOS doesn't use IAPs, for the moment, so we can safely skip them.
+precheck_include_in_app_purchases false
```

Comment:
Ported this change from the other apps `Deliverfile` setup, but didn't really get a chance to verify it's useful because of the issue above.